workloads/README.md

```diff
@@ -0,0 +1,56 @@
+## Deploying AgnosticD workloads
+
+This is a collection of scripts that help you deploy workloads on AgnosticD Openshift clusters. It includes sample configuration files for migration related workloads.
+
+### NOTE
+
+All workloads should be available in AgnosticD repository. However, the migration related workloads are yet not part of the upstream repo. Please checkout following repo to deploy migration workloads.
```

Comment:
All workloads should be available in AgnosticD repository. However, the migration related workloads are yet not part of the upstream repo. Please checkout following repo to deploy migration workloads.

---
recommend ->>

All workloads deployed by mig-agnosticd must be present in the main AgnosticD directory on your machine. However, the workloads we'll use in this README are yet not part of the _redhat-cop/agnosticd_ repo [(see PR 489)](https://github.com/redhat-cop/agnosticd/pull/489). To use these workloads before PR 489 merges, clone the following repo and set `AGNOSTICD_HOME` to the clone path: